src/Sulu/Bundle/MediaBundle/Entity/CollectionRepository.php

```diff
@@ -187,12 +188,14 @@ public function findCollections($filter = [], $limit = null, $offset = null, $so
 
             if ($parent !== null) {
                 $qb->andWhere('parent.id = :parent');
-            }
-            if ($depth !== null) {
+                $qb->setParameter('parent', $parent);
+            } elseif ($depth !== null) {
```

Comment:
Is that really exclusive? Shouldn't it be possible to have both a `parent` and a `depth`? Or is it currently too hard to implement?